json/innsyn/soker/parts/hendelse/nyStatus.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "http://json-schema.org/draft-06/schema#",
+
+	"title": "Ny status på søknaden.",
+	"javaType": "no.nav.sbl.soknadsosialhjelp.innsyn.soker.hendelse.NyStatus",
+	"type": "object",
+	"extends": {
+		"ONLY_CODEGEN$ref": "../hendelse.json"
+	},
+	"allOf": [
+		{
+			"properties": {
+				"type": {
+					"type": "string",
+					"enum": [
+						"nyStatus"
+					]
+				}
+			}
+		}
+	],
+	"properties": {
+		"status": {
+			"type": "string",
+			"description": "Det kan legges til nye statustyper i enumen, men ingen gamle kan fjernes fra valideringsskjemaet (grunnet kompatibilitet).",
+			"enum": [
+			    "MOTTATT", "UNDER_BEHANDLING"
```

Comment:
Kan den ikke være ferdigstilt?

Reply:
Det har ikke blitt avklart ennå hvordan vi håndterer at søknaden har blitt ferdigbehandlet. Mulig vi likevel bare bør definerte en kode som heter "FERDIG".

Reply:
Det er bakoverkompatibelt for leverandørene å legge til bruk av nye statuser, men det er vanskelig å fjerne gamle. Tenker vi derfor bør vente med å legge til FERDIG/AVSLUTTET/... til vi er ganske sikre på at det er noe vi vil ha.

Det skal være helt greit å merge inn til master før dette er på plass. FIKS/leverandørene kan starte utvikling selv om dette ikke er klart. Likevel bør det avklares siden vi ikke kan vise status til bruker før vi har status for FERDIG/AVSLUTTET.

Reply:
Hvis det er helt uproblematisk å legge det til senere er det bedre å vente ja :)